app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -751,10 +758,10 @@ export default {
     },
     async sendMessage(messagePayload) {
       try {
-        await this.$store.dispatch(
-          'createPendingMessageAndSend',
-          messagePayload
-        );
+        await this.$store.dispatch('createPendingMessageAndSend', {
+          data: messagePayload,
+          analytics: this.sendMessageAnalyticsData,
+        });
```

Comment:
@iamsivinI Would it be possible to keep the analytics part here instead of passing it to the store?

@scmmishra WDYT?

Reply:
@muhsin-k I think so. It was written in the store that the reason, I do it like this way.

Reply:
@muhsin-k Done https://github.com/chatwoot/chatwoot/pull/7760/commits/5e63b33ddb0be9a94dc5ed0f635d5475f9c3c5cd